offline/controller/controller.m

```diff
@@ -0,0 +1,125 @@
+function [trajectory] = controller()
+
+    addpath('../localizer/latlonutm/Codes/matlab');
+    global wheel_base
+    global velocity
+    global steering_vel
+    global dt
+
+    save_data = true;
+    wheel_base = 1.13;
+    utm_zone = 17;
+    first_heading = deg2rad(250);
+    lat_long = [40.441670, -79.9416362];
+    dt = 0.001; % 1000Hz
+    m = 50; % 20Hz
+    velocity = 8; % m/s, 17.9mph, forward velocity
+    steering_vel = deg2rad(40); % 40deg/s, reaction speed to control cmds
+
+    [x, y, ~] = ll2utm(lat_long(1), lat_long(2));
```

Comment:
`ll2utm` is actually a pretty decent naming convention, IMO. I think this is all right as is